src/coreComponents/linearAlgebra/interfaces/hypre/mgrStrategies/SinglePhasePoromechanicsConformingFracturesReservoirFVM.hpp

```diff
@@ -0,0 +1,135 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2016-2024 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2024 TotalEnergies
+ * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2023-2024 Chevron
+ * Copyright (c) 2019-     GEOS/GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file SinglePhasePoromechanicsConformingFracturesReservoirFVM.hpp
+ */
+
+#ifndef GEOS_LINEARALGEBRA_INTERFACES_HYPREMGRSINGLEPHASEPOROMECHANICSCONFORMINGFRACTURESReservoirFVM_HPP_
+#define GEOS_LINEARALGEBRA_INTERFACES_HYPREMGRSINGLEPHASEPOROMECHANICSCONFORMINGFRACTURESReservoirFVM_HPP_
+
+#include "linearAlgebra/interfaces/hypre/HypreMGR.hpp"
+
+namespace geos
+{
+
+namespace hypre
+{
+
+namespace mgr
+{
+
+/**
+ * @brief SinglePhasePoromechanicsConformingFracturesReservoirFVM strategy.
+ *
+ * dofLabel: 0 = displacement, x-component
+ * dofLabel: 1 = displacement, y-component
+ * dofLabel: 2 = displacement, z-component
+ * dofLabel: 3 = face-centered lagrange multiplier (tn)
+ * dofLabel: 4 = face-centered lagrange multiplier (tt1)
+ * dofLabel: 5 = face-centered lagrange multiplier (tt2)
+ * dofLabel: 6 = pressure (cell elem + fracture elems)
```

Comment:
Isothermal case:

```suggestion
 * dofLabel: 0 = displacement, x-component
 * dofLabel: 1 = displacement, y-component
 * dofLabel: 2 = displacement, z-component
 * dofLabel: 3 = face-centered lagrange multiplier (tn)
 * dofLabel: 4 = face-centered lagrange multiplier (tt1)
 * dofLabel: 5 = face-centered lagrange multiplier (tt2)
 * dofLabel: 6 = pressure (cell elem + fracture elems)
 * dofLabel: 7 = well pressure
 * dofLabel: 8 = well rate
```

Thermal case:

```suggestion
 * dofLabel: 0 = displacement, x-component
 * dofLabel: 1 = displacement, y-component
 * dofLabel: 2 = displacement, z-component
 * dofLabel: 3 = face-centered lagrange multiplier (tn)
 * dofLabel: 4 = face-centered lagrange multiplier (tt1)
 * dofLabel: 5 = face-centered lagrange multiplier (tt2)
 * dofLabel: 6 = pressure (cell elem + fracture elems)
 * dofLabel: 7 = temperature (cell elem + fracture elems)
 * dofLabel: 8 = well pressure
 * dofLabel: 9 = well termperature
 * dofLabel: 8 = well rate
```